client/layout/controller.js

```diff
@@ -179,6 +182,20 @@ export const getPages = () => {
 		} );
 	}
 
+	if ( window.wcAdminFeatures.profiler ) {
```

Comment:
I think this should just use the `onboarding` feature flag.

```suggestion
	if ( window.wcAdminFeatures.onboarding ) {
```